project-metrics/metrics_service/apis/travis.py

```diff
@@ -69,13 +85,25 @@ def _get_travis_token(self, github_token: Text) -> Text:
   @property
   def repo(self) -> agithub_base.IncompleteRequest:
     """Returns a partial Travis request for the repository in env.yaml."""
-    return self.repos[env.get('GITHUB_REPO')]
-
-  def fetch_builds(self, after_number: int = None) -> Dict[Text, Any]:
+    return self['repo'][env.get('GITHUB_REPO').replace('/', '%2F')]
+
```

Comment:
Can you use `encodeURIComponent` here?

Reply:
I could! That seemed heavy-weight given the known format of the repo string, since all other characters allowed in the slug don't need any encoding. I don't feel too strongly either way

Reply:
Magic strings aren't all that bad, and I've been known to use them myself. I think for this case it just makes it a little bit more readable to use a pre-existing function.

"quoting text" has a long history in web for meaning what "escape" usually means in other places, so that's fine. Turns out I was thinking of JS (`encodeURIComponent` is a built-in JS function), so you can't use that here 😝

Yes, use `urllib`…

Reply:
Yeah I'm familiar with the JS function and I think the naming is way better/clearer than `quote`. I'll swap this out then.

Reply:
Done